pkg/rbac/userData.go

```diff
@@ -408,9 +409,30 @@ func (shared *SharedData) isClusterScoped(kindPlural, apigroup string) bool {
 	// lock to prevent checking more than one at a time and check if cluster scoped resources already in cache
```

Comment:
- Can we move this function to `sharedData.go` ?
- I'm a little hesitant about hardcoding this list. It's ok for now, but we'll need to revisit it later.